base/server/python/pki/server/deployment/__init__.py

```diff
@@ -1451,8 +1451,7 @@ def get_tps_connector(self, instance, subsystem):
             '-n', nickname,
             'tks-tpsconnector-show',
             '--host', self.mdict['pki_hostname'],
-            '--port', securePort,
-            '--output-format', 'json'
+            '--port', securePort
```

Comment:
This code is calling `pki tks-tpsconnector-show` instead of `pki kra-key-recover`, so maybe we want to leave it for a different PR.

Reply:
Yeah good point, I'll reinstate.